# HG changeset patch
# User Stas Boukarev <stassats@gmail.com>
# Date 1491011923 -10800
#      Sat Apr 01 04:58:43 2017 +0300
# Node ID 1065bfec0105c17816ebc2715583e7707892da61
# Parent  7f3149adfcff6e64ceba1f43a056ab65431fbcbd
Fix float state saving on x86-64 Darwin without AVX.

Fixes lp#1678347.

diff -r 7f3149adfcff -r 1065bfec0105 src/runtime/x86-64-darwin-os.c
--- src/runtime/x86-64-darwin-os.c	Sat Apr 01 04:25:21 2017 +0300
+++ src/runtime/x86-64-darwin-os.c	Sat Apr 01 04:58:43 2017 +0300
@@ -300,7 +300,7 @@
     x86_avx_state64_t float_state;
     mach_msg_type_number_t float_state_count = avx_supported? x86_AVX_STATE64_COUNT : x86_FLOAT_STATE64_COUNT;
     x86_avx_state64_t *target_float_state;
-    int float_state_flavor = avx_supported? x86_AVX_STATE64 : x86_FLOAT_STATE64_COUNT;
+    int float_state_flavor = avx_supported? x86_AVX_STATE64 : x86_FLOAT_STATE64;
 #else
     x86_float_state64_t float_state;
     mach_msg_type_number_t float_state_count = x86_FLOAT_STATE64_COUNT;
